test/src/tests.coffee

```diff
@@ -796,7 +796,7 @@ asyncTest 'Controller: showPaletteInTextMode false', ->
     strictEqual aceEditor.style.left, '0px'
     editor.performFreezeAnimation 10, 10, ->
       strictEqual paletteWrapper.style.left, '0px'
-      strictEqual aceEditor.style.left, '-9999px'
+      strictEqual aceEditor.style.left, '0px'
```

Comment:
`-9999px` is usually used to hide something off the screen. @cpirich are you confident that we are not accidentally showing something that should be hidden after commit 07865f2 ?